Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [package]
+wayland-sys = { git  = "https://github.com/Smithay/wayland-rs.git", optional = true }
```

Comment:
Looks like this line has nothing to do here, is it a failpaste?

Reply:
yep, indeed